models/nexusmutual/ethereum/nexusmutual_ethereum_capital_pool_erc20_transactions.sql

```diff
@@ -0,0 +1,99 @@
+
+{{ config(
+    alias ='capital_pool_erc20_transactions',
+    partition_by = ['block_time'],
```

Comment:
`block_time` is a bit too granular to partition by, we'd want larger chunks of data. maybe by day? week? month?